### PR TITLE
Added pymagewell recipe

### PR DESCRIPTION
**Pull request by crnbaker**
[pymagewell](https://github.com/KCL-BMEIS/pymagewell) is a Python library for interfacing with Magewell ProCapture frame grabbers.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by crnbaker** [comment]
conda-forge-admin, please ping team

**Comment by crnbaker** [comment]
Ready for review, thanks in advance.

@conda-forge-admin, please ping team

**Comment by crnbaker** [comment]
Pinging again because this PR has rolled off the first page.

@conda-forge-admin, please ping team

File: recipes/pymagewell/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{% set owner = "KCL-BMEIS" %}
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:1]: You don't need to define a variable that is used only once. Also, if you hardcode this info below, the URL become more readable and folks can click on it.

File: recipes/pymagewell/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    sha256: bc906d1c5c70146d4131ff8932d969279c2a97f3f8d8792ba6093b2b1bd972c9
+  - url: https://github.com/{{ owner }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4a136f5972d6e35fa01663a4e38ea4811b2d5918c03f0e9254ddb41d41a14fd2
+    folder: test-data
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<39]
+  skip: True  # [linux]
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:21]:
```suggestion
  skip: True  # [py<39 or linux]
```

File: recipes/pymagewell/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    - mwcapture
+    - pymagewell
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - test-data/tests
+  commands:
+    - pip check
+    - pytest  # [win]
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:45]: This is already a Windows only pkg, no need for the pre-processing selector.
```suggestion
    - pytest  # [win]
```

```suggestion
    - pytest
```

File: recipes/pymagewell/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  skip: True  # [py<39]
+  skip: True  # [linux]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.21.4
+    - pywin32 >=303  # [win]
```
[the remark this answers is not in the document]
Comment (crnbaker) [reply recipes/pymagewell/meta.yaml:23]: The packae can also be run in a mock mode on macOS, and this doesn't require pywin32

**Comment by crnbaker** [comment]
Thanks for the review @ocefpaf. I've made some of the requested changes but have some questions about the platform selectors (see comments).

File: recipes/pymagewell/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+package:
+  name: pymagewell 
+  version: 0.1.3 
+
+source:
+  - url: https://pypi.io/packages/source/p/pymagewell/pymagewell-0.1.3.tar.gz
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:6]:
```suggestion
  - url: https://pypi.io/packages/source/p/pymagewell/pymagewell-{{ version }} .tar.gz  
```

File: recipes/pymagewell/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
+package:
+  name: pymagewell 
+  version: 0.1.3 
+
+source:
+  - url: https://pypi.io/packages/source/p/pymagewell/pymagewell-0.1.3.tar.gz  
+    sha256: bc906d1c5c70146d4131ff8932d969279c2a97f3f8d8792ba6093b2b1bd972c9
+  - url: https://github.com/KCL-BMEIS/pymagewell/archive/refs/tags/v0.1.3.tar.gz
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:8]:
```suggestion
  - url: https://github.com/KCL-BMEIS/pymagewell/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/pymagewell/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,53 @@
+package:
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:1]:
```suggestion
{% set version = "0.1.3" %}

package:
```

File: recipes/pymagewell/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+  name: pymagewell 
+  version: 0.1.3
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:3]:
```suggestion
  version: {{ version }}
```

**Review by ocefpaf — COMMENTED** [review]
That will allow the automated infrastructure we have to update the package.

File: recipes/pymagewell/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
+
```
Comment (ocefpaf) [inline recipes/pymagewell/meta.yaml:1]: Can you remove this extra line?

**Pull request merged**